.github/.ubiquibot-config.yml

```diff
@@ -0,0 +1,6 @@
+plugins:
+  - name: test-app
+    id: test-app
+    uses:
+      - plugin: https://ubiquibot-issue-comment-embeddings.sshivaditya.workers.dev
+        runsOn: [ "issue_comment.created", "issue_comment.edited", "issue_comment.deleted" ]
```

Comment:
Can't we do `"issue_comment"` to be concise? @gentlementlegen

Reply:
Not with the current implementation because it wouldn't match the event and wouldn't be triggered.

Reply:
This file should be removed, otherwise no plugin will run inside of this repo.